Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: rnshah9/mayhem-felang
+target: fe
+
+cmds:
+  - cmd: /repo/fe @@
+    filepath: /test.fe
```

Comment:
Please genericize the project name. Also, I believe you may be using the `filepath` directive incorrectly. To include a corpus, please use the `testsuite` directive. https://mayhem.forallsecure.com/docs/reference/mayhemfile/#testsuite